test/test.ts

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs");
+
+const wasmBuffer = fs.readFileSync("./index.wasm");
+
+WebAssembly.compile(wasmBuffer).then((wasmModule) => {
+    console.log(wasmModule.toString());
+    WebAssembly.instantiate(wasmModule).then((instance) => {
+        const lib = instance.exports;
+        lib.testCosmos();
+        lib.testIbc();
+        console.log("Test succesfull!!");
+    });
+});
```

Comment:
This looks good! Could we a testing framework here? Something lightweight like [Jest](https://jestjs.io/).